packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx

```diff
@@ -81,7 +85,7 @@ function CardCategory({
   return (
     <CardLayout
       href={href}
-      icon="🗃️"
+      icon={startWithEmoji(item.label) ? "" : "🗃️"}
```

Comment:
Not a super fan of this logic.

Those things are different meaning:

```jsx
<CardLayout icon="">🔌 Routing</CardLayout>
<CardLayout icon="🔌">Routing</CardLayout>
```

We probably shouldn't assume the icon/children will always be rendered inline. We may change the layout later, or users can swizzle it, and decide to render the icon in a separate dom element. 

Do we really want both cases to diverge and possibly render different things?